src/main/java/frc/robot/subsystems/NavXSubsystem.java

```diff
@@ -0,0 +1,49 @@
+/*----------------------------------------------------------------------------*/
+ /* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+ /* Open Source Software - may be modified and shared by FRC teams. The code   */
+ /* must be accompanied by the FIRST BSD license file in the root directory of */
+ /* the project.                                                               */
+ /*----------------------------------------------------------------------------*/
+package frc.robot.subsystems;
+
+import com.kauailabs.navx.frc.AHRS;
+
+import edu.wpi.first.wpilibj.SPI;
+import edu.wpi.first.wpilibj.Timer;
+import frc.robot.pose.NavXData;
+
+/**
+ *
+ * @author dcowden
+ */
+public class NavXSubsystem extends BaseSubsystem {
+
+    private final AHRS navX = new AHRS(SPI.Port.kMXP);
+    private Timer timer;
+    private double TIMEOUT_CALIBRATION_SECONDS = 31;
```

Comment:
good idea using a constant, but lets not let it go 31 seconds. thats too long. more like 5 or so